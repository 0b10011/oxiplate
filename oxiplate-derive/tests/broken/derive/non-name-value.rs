use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate("foo")]
struct Data;

fn main() {
    print!("{}", Data);
}