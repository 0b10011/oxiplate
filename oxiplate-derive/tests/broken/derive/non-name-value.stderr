error: Incorrect syntax for external template. Should look something like:
       #[oxiplate = "/path/to/template/from/templates/directory.txt.oxip"]
 --> tests/broken/derive/non-name-value.rs:4:1
  |
4 | #[oxiplate("foo")]
  | ^^^^^^^^^^^^^^^^^^
